tools/pipelines/templates/include-publish-npm-package.yml

```diff
@@ -47,6 +47,14 @@ stages:
       buildToolsVersionToInstall: ${{ parameters.buildToolsVersionToInstall }}
       pnpmStorePath: ${{ parameters.pnpmStorePath }}
 
+  # Only generate manifest files for runs in the internal project (i.e. CI runs, not PR run), for the main, test and release branch, and
+  # for the build of the client release group (we don't need manifests for anything else).
+  # Enabling this template for every PR run risks overwriting existing manifest files uploaded to Azure blobs. Therefore, it's crucial to restrict this template to commits merged into the main and release branch.
```

Comment:
:( We aren't going to be able to detect incompatibility before merge to main or release branch?
I guess with release branch we are expecting to be more careful and not as much of a concern.
One option to consider, if we can support it, is to have PRs use a keyword in the description that indicates a desire to check for compat.
Having a manual option is good! Just know that if it isn't easy and in your face, we humans are going to do it wrong. ;)